src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs

```diff
@@ -288,6 +293,10 @@ private async Task ApplySchemaInternalAsync(int version, string script, bool app
         await _schemaManagerDataStore.ExecuteScriptAndCompleteSchemaVersionAsync(script, version, applyFullSchemaSnapshot, cancellationToken).ConfigureAwait(false);
 
         _logger.LogInformation("Schema migration completed successfully for the version : {Version}.", version);
+
+        // this notification is used by the fhir service to initialized the dictionaries
```

Comment:
Might be better to comment what this event is for and why its useful

Reply:
Sure, updated the comment